Directory.Build.props

```diff
@@ -38,4 +38,7 @@
     <AdditionalFiles Include="$(MSBuildThisFileDirectory)\codeAnalysis\StyleCop.json" Link="StyleCop.json" />
   </ItemGroup>
 
+  <ItemGroup>
+	  <RuntimeHostConfigurationOption Include="System.Runtime.Loader.UseRidGraph" Value="true" />

```

Comment:
I remember having to add this when I tried to move to .net8.0.  What is the reason for this again, and can you add a comment explaining it.